spec/repository_spec.rb

```diff
@@ -41,10 +41,6 @@
         )
       end
 
-      it 'installs gnupg' do
-        expect(chef_run).to install_package('gnupg')
-      end
-
```

Comment:
We still install `gnupg` on some situations so we want to keep the spec test. One way to reflect what you did in the code is:

```
      it 'installs gnupg' do
        expect(chef_run).to install_package('gnupg') if chef_run.node['packages']['gnupg2'].nil?
      end
```

Would you mind adding back the spec tests following that pattern?

Reply:
Of course... will do that in the evening...

Reply:
Done for unit test

Reply:
Awesome, thanks for taking the time to fix that!